Serializer/Normalizer/AnnotationNormalizer.php

```diff
@@ -16,15 +16,20 @@
 use Superbrave\GdprBundle\Annotation\AnnotationReader;
 use Superbrave\GdprBundle\Manipulator\PropertyManipulator;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 
 /**
  * Normalizes object data based on the specified property annotation.
  *
  * @author Niels Nijens <nn@superbrave.nl>
+ * @author Jelle van Oosterbosch <jvo@superbrave.nl>
  */
-class AnnotationNormalizer implements NormalizerInterface
+class AnnotationNormalizer implements NormalizerInterface, NormalizerAwareInterface
```

Comment:
Is die NormalizerAwareInterface nog nodig?

Reply:
Yes the NormalizerAwareInterface is required. If you look inside the `IterableNormalizer::normalize` or the `AnnotationNormalizer::normalize` you can see that the inner `$this->normalizer` is being called.